1_fetch.R

```diff
@@ -46,9 +47,39 @@ p1_targets_list <- list(
   tar_target(
     p1_nhd_reaches,
     download_nhdplus_flowlines(p1_drb_comids_all_tribs$COMID)
-  )
-
+  ),
+  
+  # STATSGO SOIL Characteristics
+  ## get selected child items nhdv2 STATSGO Soil Characteristics
+  ## 1) Text attributes and 2) Layer attributes
+  tar_target(
+    p1_selected_statsgo_sbid_children,
+    item_list_children(sb_id = nhd_statsgo_parent_sbid) %>% 
```

Comment:
```suggestion
    sbtools::item_list_children(sb_id = nhd_statsgo_parent_sbid) %>% 
```

I see that you use this format in the next target, so for consistency I suggested the same here.